packages/angular/projects/ui-angular/src/lib/components/authenticator/components/confirm-reset-password/amplify-confirm-reset-password.component.html

```diff
@@ -10,7 +10,6 @@ <h3 class="amplify-heading">{{ headerText }}</h3>
     </amplify-slot>
     <amplify-form-field
       name="confirmation_code"
-      type="number"
```

Comment:
Avoiding explicit `type` assignment in favor of default `type` set from `@aws-amplify/ui` here: https://github.com/aws-amplify/amplify-ui/blob/093c08399eb912c7359883a936cfcdb314a8d2d1/packages/ui/src/helpers/authenticator/form.ts#L19-L24